pkg/apis/monitoring/v1/thanos_types.go

```diff
@@ -191,6 +191,14 @@ type ThanosRulerSpec struct {
 	// This is an alpha field and requires enabling StatefulSetMinReadySeconds feature gate.
 	// +optional
 	MinReadySeconds *uint32 `json:"minReadySeconds,omitempty"`
+	// AlertRelabelConfig configures alert relabelling in ThanosRuler.
+	// Alert relabel configurations specified must have the form as specified in the official Prometheus documentation:
+	// https://prometheus.io/docs/prometheus/latest/configuration/configuration/#alert_relabel_configs
+	// Alternative to AlertRelabelConfigFile, and lower order priority.
+	AlertRelabelConfig *v1.SecretKeySelector `json:"alertRelabelConfig,omitempty"`
```

Comment:
(nit) to be consistent with the Prometheus CRD.

```suggestion
	AlertRelabelConfigs *v1.SecretKeySelector `json:"alertRelabelConfigs,omitempty"`
```